triggers/simulations/overview.rst

```diff
@@ -0,0 +1,461 @@
+    TriggerTimer.singleMessage(5.seconds, 2.seconds)(breedingTrigger)
+
+.. note::
+  Currently, there is no support for extracting and using the Daml trigger ``heartbeat`` expression when scheduling heartbeat messages. This is a known limitation.
+
+Filtering Ledger Transaction Messages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, trigger processes will receive all ledger create and archive events for the templates that they have registered for. Sometimes, it might be useful to have more granular control over which events a trigger process may receive - the ``TriggerFilter.apply`` wrapper function provides this control within a simulation.
+
+For example, to have a trigger process ignore transaction messages with an effective date that is too old (e.g. older than a ``lifeTime: FiniteDuration``), we could use:
+
+.. code-block:: scala
+  object LifeTimeFilter {
+    def apply(
+      lifeTime: FiniteDuration
+    )(
+      consumer: ActorRef[TriggerProcess.Message]
+    ): Behavior[TriggerProcess.Message] = {
+      val lifeTimeFilter = { case TriggerMsg.Transaction(transaction) =>
+  	    val effectiveAt = transaction.effectiveAt.seconds
+  	    val now = System.currentTimeMillis / 1000
+
+  	    now <= effectiveAt + lifeTime.toSeconds
+      }
+
+      TriggerFilter(lifeTimeFilter)(consumer)
+    }
+  }
+
+.. note::
+  Currently, trigger filtering can not be implemented in Daml in a manner that impacts the trigger view of the ACS.
+
+Preserving Simulation Metrics for Offline Analysis
+--------------------------------------------------
+
+Reporting processes are implemented as Akka actors. They are (automatically) created as child processes of a ledger process and used to collect:
+
+- trigger metric data
+- trigger resource usage data
+- information about the difference between the trigger and ledger contract store views.
+
+Collected reporting data is saved into CSV files - the precise location of which is configured by overriding the ``simulationConfig: TriggerSimulationConfig`` implicit. For example:
+
+.. code-block:: scala
+  class ExampleSimulation extends TriggerMultiProcessSimulation {
+
+    override protected implicit lazy val simulationConfig: TriggerSimulationConfig =
+      TriggerSimulationConfig(
+        triggerDataFile = Paths.get("/data/trigger-simulation-metrics-data.csv"),
+        acsDataFile = Paths.get("/data/trigger-simulation-acs-data.csv"),
+      )
+
+    override protected def triggerMultiProcessSimulation: Behavior[Unit] = ???
+  }
+
+If explicit file paths are configured for the reporting data, then a simple ``bazel test`` should be sufficient for running the simulation and saving the reporting data (in the configured output files).
+
+By default though, all reporting data is stored within the bazel run directory and so, after a simulation test run has completed will be automatically deleted. To preserve the simulation reporting data then a ``bazel test --test_tempdir=/tmp`` or similar should be used.
```

Comment:
`Though` is an overloaded and colloquial word, we can replace it with `however` to make our intent more obvious.
`By default though,` -> `By default, however,`